packages/math-input/src/components/keypad/navigation-pad.tsx

```diff
@@ -0,0 +1,60 @@
+import Color from "@khanacademy/wonder-blocks-color";
+import {View} from "@khanacademy/wonder-blocks-core";
+import {StyleSheet} from "aphrodite";
+import * as React from "react";
+
+import Keys from "../../data/key-configs";
+
+import NavigationButton from "./navigation-button";
+
+import type {ClickKeyCallback} from "../../types";
+
+export type Props = {
+    onClickKey: ClickKeyCallback;
+};
+
+export default function NavigationPad(props: Props) {
+    const {onClickKey} = props;
+
+    return (
+        <View style={styles.container}>
+            <View style={styles.grid}>
+                <NavigationButton keyConfig={Keys.UP} onClickKey={onClickKey} />
```

Comment:
**thought**: I read through `NavigationButton` and saw that we define the row/col in that component. That feels a bit off for each button to lay itself out. What do you think about defining the grid row/col as a prop here?  That would make it easy for future folks to change the layout here, in the container, rather than needing to go look at the button itself. I guess the summary is that we split layout between this container (defining the `gredtemplateXyz` stuff here) and the buttons (the specific row/col).